### ISSUE TITLE
Features Ideas!!

### ISSUE THREAD

**Issue by DavidxRussell** [trevor99z/Reinforcement#10]
Hey bro! Following up from our message via Spigot.

I think the project should lean towards Lodestones working as radius reinforcement. By right-clicking the Lodestone, a user could access a GUI. There they could add trusted members, upgrade reinforcement (simply by adding more obsidian, diamonds, etc to the Lodestone), expand reinforcement region, and also access block history. I think block history would be dope for dynamics as users could see what enemies broke at what time so they’re prepared to fight back.

If someone destroys a Lodestone the region would start to “decay” by the reinforcement dropping breaks. This would create a dynamic where users keep their Lodestone hidden, laced in obsidian and buried deep underground. Griefers would target the Lodestone to make their raid easier. 
When a lodestone is being tampered with the owner should be notified. Would be dope if we could do a discord hook, so the rest of server would be ready to jump in to the fight even if they’re offline.

GriefPrevention is lacking and reinforcement is the move!

TLDR;
Lodestone Radius Reinforcement
     - GUI
     - Members
     - Block History
     - Decay
     - Discord Hooks

**Comment by trevor99z** [comment]
Thanks for the feature suggestions! I will look at adding these when I have some time.
